packages/doenetml/rust/src/Viewer/PageViewer.jsx

```diff
@@ -719,9 +719,9 @@ export default function PageViewer(props) {
         console.log(`send message to create core ${pageNumber}`);
 
         coreWorker.current = new Worker(
-            new URL("doenetml-worker/CoreWorker.js", window.location),
+            new URL("doenetml-worker/CoreWorker.iife.js", window.location),
             {
-                type: "module",
+                type: "classic",
```

Comment:
I don't think this file should be touched at all. Everything in `rust/` should be ignored.

Reply:
Agreed, I tried to back all of the changes in the rust package out after I realized I had been a little too liberal with my find/replace. I was planning on leaving comments near these two constructors to warn us that we probably want to do similar refactoring when we go back into the rust version.